### PR TITLE
Feature/1514 load from xls

### PR DESCRIPTION
**Pull request by Mergoth**

### CODE REVIEW

File: classificators/src/main/java/ru/okpdmarket/fake/FakeClassificatorDaoImpl.java [hunk 1]
```diff
@@ -45,66 +44,66 @@ public boolean exists(String s) {
         classificator.setCode("okpd");
         classificator.setName("ОКПД");
         classificator.setDescription("тестовый ОКПД");
-        classificator.setTree(new ArrayList<>());
+        // classificator.setTree(new ArrayList<>());
```
Comment (Vladlis) [inline classificators/src/main/java/ru/okpdmarket/fake/FakeClassificatorDaoImpl.java]: Do not comment any code in this and other files.

File: classificators/src/main/java/ru/okpdmarket/dao/ClassificatorItemsDao.java [hunk 2]
```diff
@@ -0,0 +1,13 @@
+package ru.okpdmarket.dao;
+
+import org.springframework.data.repository.CrudRepository;
+import org.springframework.stereotype.Repository;
+import ru.okpdmarket.dao.dto.ClassificatorItemDaoDto;
+
+/**
+ * Created by lalka on 10/26/2016.
```
Comment (Vladlis) [inline classificators/src/main/java/ru/okpdmarket/dao/ClassificatorItemsDao.java]: Remove this.

**Review by Vladlis — DISMISSED** [review]
Why commits for different task are submitted in one PR?

File: classificators/src/main/java/ru/okpdmarket/dao/ClassificatorItemsDao.java (hunk 2 above)
Comment (Mergoth) [reply classificators/src/main/java/ru/okpdmarket/dao/ClassificatorItemsDao.java]: ok!

File: classificators/src/main/java/ru/okpdmarket/fake/FakeClassificatorDaoImpl.java (hunk 1 above)
Comment (Mergoth) [reply classificators/src/main/java/ru/okpdmarket/fake/FakeClassificatorDaoImpl.java]: done

**Comment by Vladlis** [comment]
1) please address one more question from previous review - Why commits for different task are submitted in one PR?
2) generally ok, but I'd move clean up in a different PR and changed commit message as ticket 1514 is not relevant
3) what about codecov?

**Review by Vladlis — DISMISSED** [review]
Please address codacy issues and consider squashing commits to avoid multiple changes in the same areas of code for one task.
Everything else looks good.

**Pull request merged**